discovery/kubernetes/kubernetes_test.go

```diff
@@ -320,3 +322,16 @@ func TestFailuresCountMetric(t *testing.T) {
 		})
 	}
 }
+
+func TestDeletedNodeName(t *testing.T) {
+	node := &apiv1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+	}
+
+	// The key would probably be the
```

Comment:
something missing?

Reply:
`The key would probably be the node's name` :) But because one can override the `keyFunc` and to be immune to that I thought it's better to look at `.Obj`